src/Application.js

```diff
@@ -56,7 +58,9 @@ const stateDefaults = {
   [timersScale]: 0.5,
   [inputCount]: 0.5,
   [keyBinding]: {},
-  [colorVision]: 'none'
+  [colorVision]: 'none',
+  [fullScreen]: false,
+  [indexedDBPlugin]: false
```

Comment:
We don't actually need to include the `indexedDBPlugin` in the feature lists. Because its part of the `UserData` class its not actually a game feature, but something that the dev will use directly. This will go for any of the references to it in `Application`